sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1310,6 +1304,20 @@ class HistGradientBoostingRegressor(RegressorMixin, BaseHistGradientBoosting):
         "quantile",
     )
 
+    _parameter_constraints = {**BaseHistGradientBoosting._parameter_constraints}
+    _parameter_constraints.update(
+        {
+            "loss": [
+                StrOptions(
+                    set(_VALID_LOSSES),
```

Comment:
This is the only place the class attribute _VALID_LOSSES is used. I think we can remove it and directly put the set of valid losses.

Reply:
Do you mean like the following:
```python
"loss": [ StrOptions({
        "squared_error",
        "absolute_error",
        "poisson",
        "quantile"
})
```

Reply:
yes